### PR TITLE
 NXP-28838 explorer json export (Part 2/2)

### PR DESCRIPTION
**Pull request by atchertchian**
Rest & UI follow-up of #12

### CODE REVIEW

File: modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/adapters/OperationInfoDocAdapter.java
Comment (efge) [inline modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/adapters/OperationInfoDocAdapter.java:203]: You can use `StringUtils.isNotEmpty`

File: modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/introspection/RuntimeSnapshot.java
Comment (atchertchian) [reply modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/introspection/RuntimeSnapshot.java:409]: :+1: i went for "isNotBlank" still, given past inconsistencies on this information persistence

File: modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/snapshot/TargetExtensionPointSnapshotFilter.java
Comment (atchertchian) [reply modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/snapshot/TargetExtensionPointSnapshotFilter.java:62]: :+1: i actually used these later on and forgot to replace this one

**Comment by atchertchian** [comment]
Force-pushed to resolve conflicts after merge of #26

**Review by efge — APPROVED** [review]

**Comment by atchertchian** [comment]
Will need to force-push again to resolve conflict after merge of #29, so will merge as is right after.

Will probably open a last PR on this issue to prevent referencing requirements that would not be part of a partial distribution.

**Pull request merged**
